tvm/src/codegen/opencl/codegen_aocl.cc

```diff
@@ -140,7 +140,7 @@ void CodeGenAOCL::PrintType(Type t, std::ostream &os)
       if(t.is_uint()) {
         switch(t.bits()) {
           case 32:
-            os << "cl_uint";
+            os << "uint";
```

Comment:
Why is the change? Do we also need to change the below case?

Reply:
The previous should work. But there is an error from AOC saying that it failed to find the deceleration of `cl_uint`.